cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage/Config/MigrateConsumedPruneTxOut.hs

```diff
@@ -54,9 +53,31 @@ import Test.Cardano.Db.Mock.UnifiedApi (
 import Test.Cardano.Db.Mock.Validate (assertBlockNoBackoff, assertEqQuery, assertTxCount, assertTxInCount, assertTxOutCount, assertUnspentTx, checkStillRuns)
 import Test.Tasty.HUnit (Assertion)
 
+------------------------------------------------------------------------------
+-- Custom Configs
+------------------------------------------------------------------------------
+configTxOutPruneForceTxIn :: SyncNodeConfig -> SyncNodeConfig
+configTxOutPruneForceTxIn cfg = do
+  cfg {dncInsertOptions = (dncInsertOptions cfg) {sioTxOut = TxOutPrune (ForceTxIn True)}}
+
+configTxOutPrune :: SyncNodeConfig -> SyncNodeConfig
+configTxOutPrune cfg = do
+  cfg {dncInsertOptions = (dncInsertOptions cfg) {sioTxOut = TxOutPrune (ForceTxIn False)}}
+
+configConsume :: SyncNodeConfig -> SyncNodeConfig
+configConsume cfg = do
+  cfg {dncInsertOptions = (dncInsertOptions cfg) {sioTxOut = TxOutConsumed (ForceTxIn False)}}
+
+configBootstrap :: SyncNodeConfig -> SyncNodeConfig
+configBootstrap cfg = do
+  cfg {dncInsertOptions = (dncInsertOptions cfg) {sioTxOut = TxOutBootstrap (ForceTxIn False)}}
+
+------------------------------------------------------------------------------
+-- Tests
+------------------------------------------------------------------------------
 txConsumedColumnCheck :: IOManager -> [(Text, Text)] -> Assertion
 txConsumedColumnCheck = do
-  withCustomConfigAndDropDB cmdLineArgs Nothing babbageConfigDir testLabel $ \interpreter mockServer dbSyncEnv -> do
+  withCustomConfigAndDropDB cmdLineArgs (Just configTxOutPruneForceTxIn) babbageConfigDir testLabel $ \interpreter mockServer dbSyncEnv -> do
```

Comment:
The file used was `test-db-sync-config-consumed.json` which assumes only consumed is used and not pruned. Also these config functions could be shared between Conway and Babbage. How about moving them to `Test.Cardano.Db.Mock.Config`?

Reply:
lol couldn't work out what was happening as it's all changed locally. The files weren't saved 🤦 
But yeah I'll move the configs to `Test.Cardano.Db.Mock.Config` 👍